src/shimoku/actions_execution/execute_action.py

```diff
@@ -345,35 +356,51 @@ def execute_action(code: str, print_code: bool = False):
             f"asyncio.get_event_loop().create_task(main_{main_uuid}())",
         )
     else:
+        script_code_lines.insert(4, "")
+        script_code_lines.insert(4, "    js_access_token = None")
+        script_code_lines.insert(4, "    js_universe_id = 'local'")
+        script_code_lines.insert(4, "    js_environment = 'production'")
+        script_code_lines.insert(4, "    js_workspace_id = 'local'")
+
+        script_code_lines.insert(
+            -3, "    shimoku_client._async_pool.ACTIONS_TEST = True"
+        )
         script_code_lines.append(
             f"asyncio.run(main_{main_uuid}())",
         )
 
     if print_code:
         for i, line in enumerate(script_code_lines):
             print(f"{i:3} | {line}")
-
-    script_ast = ast.parse("\n".join(script_code_lines))
-
-    analyzer = ActionsChecker()
-    analyzer.visit(script_ast)
-    if not analyzer.exists_action_function:
-        log_error(logger, "Action function is not defined.", ActionError)
-    if not analyzer.shimoku_imported:
-        log_error(logger, "Shimoku's Client is not imported.", ActionError)
-    modifier = AwaitInserter(analyzer.current_scope)
-    modifier.visit(script_ast)
-    # dump = ast.dump(script_ast, indent=4).split("\n")
-    # for i, line in enumerate(dump):
-    #     print(f'{i:3} | {line}')
+    try:
+        script_ast = ast.parse("\n".join(script_code_lines))
+        analyzer = ActionsChecker()
+        analyzer.visit(script_ast)
+        if not analyzer.exists_action_function:
+            log_error(logger, "Action function is not defined.", ActionError)
```

Comment:
Errors messages should be placed in an external file and imported here, that is a way to prevent duplication of messages

Reply:
Should i apply this to the whole codebase? I don't recall any error beign repeated, but if it is a best practice i will do it, maybe it needs it own task as it would muddy up this PR

Reply:
Better another task, I will open it